backend/pom.xml

```diff
@@ -44,6 +47,14 @@
             <version>4.12.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-security</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+        </dependency>
```

Comment:
For what do you need this dependency? JUnit is integrated in Spring Boot

Reply:
Thanks! You're right, I don't need this. I deleted it